test/itkFeatureAggregatorTest1.cxx

```diff
@@ -58,9 +58,9 @@ class FeatureAggregatorSurrogate : public FeatureAggregator<NDimension>
   }
 
 private:
-  FeatureAggregatorSurrogate(const Self &); // purposely not implemented
+  FeatureAggregatorSurrogate(const Self &) = delete; // purposely not implemented
```

Comment:
We should put `ITK_DISALLOW_COPY_AND_ASSIGN` in public section.

Reply:
@mseng10 is working on that. There are about 300 cases where this needs to be done.